packages/contracts/src/dollar/libraries/Constants.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
+import "abdk/ABDKMathQuad.sol";
```

Comment:
I was under the impression that the later versions of Solidity (including the one in this file pragma) makes ABDK library obsolete?

Reply:
@zgorizzo69 created that contract. Should be able to provide more info.

Reply:
Nop abdk is for floating point math which doesn't exist in solidity